lib/redis.js

```diff
@@ -131,8 +131,10 @@ module.exports = CoreObject.extend({
   _deploySuccessMessage: function(revisionKey) {
     var success       = green('\nUpload successful!\n\n');
     var uploadMessage = white('Uploaded revision: ')+green(revisionKey);
+    var exampleCommand = white('To activate, execute: \n\n')+
+      green('ember deploy:activate -r ' + revisionKey + '-e ' + this.config.environment);
```

Comment:
You could patch this in the configuration-reader of ember-cli-deploy and pass the environment there but this is kind of messy and will be fixed via the upcoming changes in ember-cli-deploy itself.

This would be a great addition to ember-deploy-redis. Thanks for catching this. I'll leave this open and ping you when ember-cli-deploy is ready.